src/components/Calculator.js

```diff
@@ -2,6 +2,7 @@
 import React, { useState } from 'react';
 import './Calculator.css';
 import calculate from './logic/calculate';
+import Quote from './quote';
```

Comment:
**You did great job! Project is working as expected!**

- [ ] However, in this project using [eslint-disabler](https://github.com/Elhambasir/math-magician/pull/4/files#diff-2e862d0b88bdd42eb13f6528bc2ef48a56500daca0350696b337c7f2127f4f0b:~:text=/*%20eslint%2Ddisable%20react/prop%2Dtypes%20*/) is not allowed. Kindly remove it. Thank you :)

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.